deps-check.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+function fail() {
+	printf $txtred%s$txtnon\\n "$@"
```

Comment:
This won't work as you expect. See the difference:

```
> f() {
printf '%s\n' "$@"
}
> f a b c
a
b
c
> f2() {
printf '%s\n' "$*"
}
> f2 a b c
a b c
```

Use "$*" instead of "$@".